pyiron/vasp/volumetric_data.py

```diff
@@ -69,6 +69,8 @@ class with very minor modifications. The new parser is faster
             normalize (boolean): Flag to normalize by the volume of the cell
 
         """
+        if os.stat(filename).st_size == 0:
```

Comment:
To have the same behaviour when parsing empty files we should also raise a warning here:
`s.logger.warning("File:" + filename + "seems to be corrupted/empty")`

Reply:
Fixed [here](https://github.com/pyiron/pyiron/pull/185/commits/8698b6641ca5b311f288b3a886e691eb69b0225f)